src/com/lilithsthrone/game/Properties.java

```diff
@@ -976,6 +1000,23 @@ public void loadPropertiesFromXML(){
 					}
 				}
 				
```

Comment:
Ooohhhh!!! A typo in the comments, how horrible!

Reply:
Ack, my copy pasting of code has been exposed!
I've fixed this locally, if I have another significant commit I'll push it with that, if not I'll amend it into my last commit